runtime/doc/news.txt

```diff
@@ -88,6 +88,16 @@ TREESITTER
   backwards compatibility, an option `all=false` (only return the last
   matching node) is provided that will be removed in a future release.
 
+• |vim.treesitter.language.get_filetypes()| always includes the {language}
+  argument in addition to explicitly registered filetypes.
+
+• |vim.treesitter.language.get_lang()| falls back to the {filetype} argument
+  if no languages are explicitly registered.
+
```

Comment:
Nit: it looks like the other bulleted lists in these sections do not have empty lines in-between list items